docs/Installation-Windows.md

```diff
@@ -100,12 +100,12 @@ You should see `(ml-agents)` prepended on the last line.
 
 Next, install `tensorflow`. Install this package using `pip` - which is a
 package management system used to install Python packages. Latest versions of
-Tensorflow won't work, so you will need to make sure that you install version
+TensorFlow won't work, so you will need to make sure that you install version
 1.7.1. In the same Anaconda Prompt, type in the following command _(make sure
 you are connected to the internet)_:
 
 ```sh
-pip install tensorflow==1.7.1
+pip3 install tensorflow==1.7.1
```

Comment:
@awjuliani @unityjeffrey What did we decide on regarding pip ? Do we say pip3 or do we assume people will have python3 pip ?

Reply:
based on the slack thread - we assume folks would have pip3

Reply:
however, we should remove from the windows instructions though, because we explicitly instruct them to install python a certain way